src/components/Text/Text.stories.tsx

```diff
@@ -1,18 +1,141 @@
-import { Story, Meta } from '@storybook/react';
+import type { StoryObj } from '@storybook/react';
+import clsx from 'clsx';
 import React from 'react';
 
-import { Text, Props } from './Text';
+import Text from './Text';
+import type { Color } from './Text';
+import styles from './Text.stories.module.css';
 
 export default {
-  title: 'Atoms/Text/Text',
+  title: 'Text',
   component: Text,
-} as Meta;
+  argTypes: {
+    children: {
+      control: {
+        type: 'text',
+      },
+    },
+  },
+};
 
-const Template: Story<Props> = (args) => (
-  <Text {...args}>
-    Text is a short text string rendered as either a paragraph or span.
-  </Text>
-);
+const colors = [
+  'alert',
+  'base',
+  'brand',
+  'info',
+  'inherit',
+  'neutral',
+  'success',
+  'warning',
+  'white',
+];
 
-export const Default = Template.bind({});
-Default.args = {};
+type Args = React.ComponentProps<typeof Text>;
+
+export const Body: StoryObj<Args> = {
+  args: {
+    children: 'Body paragraph 16/24',
+  },
+};
+
+export const BodySmall: StoryObj<Args> = {
+  args: {
+    size: 'sm',
+    children: 'Body small 14/20',
+  },
+};
+
+export const BodyXSmall: StoryObj<Args> = {
+  args: {
+    size: 'xs',
+    children: 'Body Xsmall 12/16',
+  },
+};
+
+export const Caption: StoryObj<Args> = {
+  args: {
+    size: 'caption',
+    children: 'Caption 12/20',
+  },
+};
+
+export const Overline: StoryObj<Args> = {
+  args: {
+    size: 'overline',
+    children: 'Overline 12/20',
+  },
+};
+
+/**
+ * 1) Used mainly for visual regression testing and to show the different color options available.
+ * 2) Has problems with snapshots since it has too many components and other stories generate enough confidence for our needs.
+ */
+export const Colors: StoryObj<Args> = {
+  render: () => {
+    const Item = ({
+      className,
+      children,
+    }: {
+      className?: string;
+      children: React.ReactElement;
+    }) => (
+      <div className={clsx(styles['color__item'], className)}>{children}</div>
+    );
+    return (
+      <div className={styles['color__table']}>
+        {colors.map((color) => (
+          <>
+            <Item
+              key={`${color}-white`}
+              className={styles['color__item-white']}
+            >
+              <Text size="h3" color={color as Color}>
+                {color}
+              </Text>
+            </Item>
+            <Item key={`${color}`} className={styles['color__item-light']}>
+              <Text size="h3" color={color as Color}>
+                {color}
+              </Text>
+            </Item>
+            <Item key={`${color}-dark`} className={styles['color__item-dark']}>
+              <Text size="h3" color={color as Color}>
+                {color}
+              </Text>
+            </Item>
+          </>
+        ))}
+      </div>
+    );
+  },
+  parameters: {
+    /* 2 */
+    axe: {
+      skip: true,
+    },
+    snapshot: {
+      skip: true,
+    },
```

Comment:
Is this one also covered by #2?

Reply:
Yea, should I also put `/* 2 */ ` there?

Reply:
do they do this style of commenting in TSX files also? I kinda like not having to look back & forth to figure out what a comment means

Reply:
Hmm, it's not part of any story file but it's pretty common amongst component files. Could just have a block comment instead